x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plugin/TextFormatTests.java

```diff
@@ -164,6 +313,24 @@ private static SqlQueryResponse emptyData() {
         );
     }
 
+    private static SqlQueryResponse withData(List<List<Object>> rows) {
+        List<ColumnInfo> headers = new ArrayList<>();
+        if (rows.isEmpty() == false) {
+            // headers
+            for (Object o : rows.get(0)) {
+                if (o instanceof Collection) {
+                    Collection<?> col = (Collection<?>) o;
+                    o = col.isEmpty() ? null : col.toArray()[0];
+                }
+
+                String typeName = fromJava(o).typeName();
+                headers.add(new ColumnInfo("index", typeName + "_column", typeName));
```

Comment:
I don't remember our take here, shouldn't the type name reveal that it's an array? something like `byte_array` or `_array` (postgres format) ?

Reply:
You mean the 3rd argument should be `typeName + "_array"`? I've updated it anyways, to be clearer, but it's not used in these tests (the formatters don't look at the type, only content).

Reply:
Thx, yep, I get, the columnInfo is not actually asserted, but it's clearer with the `_array` suffix. Thx!